### PR TITLE
[torch_glow] Implement clamp

### PR DESCRIPTION
**Pull request by bertmaher**
Test Plan: new node test, `test_clamp`

### CODE REVIEW

File: torch_glow/src/PyTorchModelLoader.cpp [hunk 1]
```diff
@@ -1209,6 +1224,35 @@ llvm::Error PyTorchModelLoader::loadAvgPool2d(const torch::jit::Node *ptNode) {
...
+
+  auto maxHandle = glow::Handle<float>::createInvalidHandle();
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      maxHandle, getGlowConstantHandle<float>(inputs[ClampInputs::max]));
+  RETURN_ERR_IF_NOT(
+      maxHandle.size() == 1,
+      glow::strFormat("max size must be 1, got %lu", maxHandle.size()));
+  auto max = maxHandle.raw(0);
+
+  auto output = F_.createClip("clip", input, min, max);
```
Comment (bertmaher) [inline torch_glow/src/PyTorchModelLoader.cpp]: Note I'm assuming min/max are constant, which PyTorch decidedly does not guarantee.  This seems like one more instance of the general caching problem so I just figured I'd not worry about it for the time being...

File: torch_glow/src/PyTorchModelLoader.cpp [hunk 2]
```diff
@@ -1209,6 +1224,35 @@ llvm::Error PyTorchModelLoader::loadAvgPool2d(const torch::jit::Node *ptNode) {
...
+  ASSIGN_VALUE_OR_RETURN_ERR(input,
+                             getGlowNodeValue(inputs[ClampInputs::input]));
+
+  auto minHandle = glow::Handle<float>::createInvalidHandle();
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      minHandle, getGlowConstantHandle<float>(inputs[ClampInputs::min]));
+  RETURN_ERR_IF_NOT(
+      minHandle.size() == 1,
+      glow::strFormat("min size must be 1, got %lu", minHandle.size()));
+  auto min = minHandle.raw(0);
```
Comment (jackm321) [inline torch_glow/src/PyTorchModelLoader.cpp]: The amount of code that this takes for each op makes me kinda want to generalize the loader to include some kinda union type also to match the inputs (like something that could be a glow nodevalue or an int or int[] etc)

File: torch_glow/src/PyTorchModelLoader.cpp (hunk 1 above)
Comment (jackm321) [reply torch_glow/src/PyTorchModelLoader.cpp]: Yeah, do not worry about it here.

File: torch_glow/tests/nodes/clamp_test.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+
+def test_clamp():
+    """Test of the PyTorch min Node on Glow."""
+
+    def test_f(x):
+        return x * torch.clamp(x + 3.0, 0.0, 6.0) * (1.0 / 6.0)
+
+    jitVsGlow(test_f, torch.randn(7), expected_fused_ops={
+              "aten::add", "aten::mul", "aten::clamp"})
```
Comment (jackm321) [inline torch_glow/tests/nodes/clamp_test.py]: why add all of these instead of just clamp?

File: torch_glow/tests/nodes/clamp_test.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+import torch
+
+from tests.utils import jitVsGlow
+
+
+def test_clamp():
+    """Test of the PyTorch min Node on Glow."""
+
+    def test_f(x):
+        return x * torch.clamp(x + 3.0, 0.0, 6.0) * (1.0 / 6.0)
```
Comment (jackm321) [inline torch_glow/tests/nodes/clamp_test.py]: what's the point of the ` * (1.0 / 6.0)`?

File: torch_glow/tests/nodes/clamp_test.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+
+from tests.utils import jitVsGlow
+
+
+def test_clamp():
+    """Test of the PyTorch min Node on Glow."""
```
Comment (jackm321) [inline torch_glow/tests/nodes/clamp_test.py]: should be clamp Node

File: torch_glow/tests/nodes/clamp_test.py (hunk 4 above)
Comment (bertmaher) [reply torch_glow/tests/nodes/clamp_test.py]: Oh lol, sorry, I wanted this for an implementation of hswish I was working on in another place, and just copied that over as a unit test.  Let me change it.

**Pull request closed without merging**